pkg/network/usm/monitor_test.go

```diff
@@ -668,17 +668,38 @@ func cleanProtocolMaps(t *testing.T, protocolName string, manager *manager.Manag
 		_, shouldOnlyZero := mapTypesToZero[mapInstance.Type()]
 
 		key := make([]byte, mapInstance.KeySize())
-		value := make([]byte, mapInstance.ValueSize())
+
+		var value any
+		var emptyValue any
```

Comment:
maybe can keep slice under shouldOnlyZero? you can find PossibleCPU outside of condition, it requires less memory then a slice

Reply:
Can you please rephrase the comment? I'm not sure i understand it

Reply:
It is about memory allocation. The map is always allocated, but is used only if shouldOnlyZero == true, sometimes allocation is not necessary. I see Huy suggested to refactor and split this function.